user-docs/latest/content-manager/writing-content.md

```diff
@@ -0,0 +1,63 @@
+| Password    | Write a password. <br><br> 💡 Click the eye icon, displayed on the right of the box, to show the password.                                                                                                                                                                                                                                                                     |
+| Enumeration | 1. Click the listbox. <br> 2. Choose an item from the list.                                                                                                                                                                                                                                                                                                                  |
+| Media       | 1. Click the media area. <br> 2. Choose an asset from the Media Library, or click the **Add more assets** button to add a new file to the Media Library. <br><br> 💡 It is possible to drag and drop the chosen file in the media area.                                                                                                                                        |
+| JSON        | Write your content, in JSON format, in the code textbox.                                                                                                                                                                                                                                                                                                                     |
+| UID         | Write a unique identifier in the textbox. A "Regenerate" button, displayed on the right of the box, allows to automatically generate a UID based on the content type name.                                                                                                                                                                                                   |
+
+### Components
+
+Components are a combination of several fields. Writing their content works exactly like for independant fields. They differ in their display, since the fields of a component are grouped together.
+
+![Writing content for a component](../assets/content-manager/edit-view_component.png)
+
+### Dynamic zones
+
+Dynamic zones are a combination of components, which themselves are composed of several fields. Writing the content of a dynamic zone requires additional steps in order to access the fields.
+
+1. Click on the ![icon add to content](../assets/content-manager/icon_add3.png) **Add to content** button.
+2. Choose a component available for the dynamic zone.
+3. Fill in the fields of the component.
+
+![Writing content for a dynamic zone](../assets/content-manager/edit-view_dynamic-zone1.png)
```

Comment:
These 2 images are a bit confusing and it's not clear on the page that are 2 different images.
I would move this image just before the procedure. And maybe keep the second after.
I'm not sure.
Or reverse the order of these 2 images.